src/components/widget.js

```diff
@@ -150,172 +145,168 @@ class Widget extends LitElement {
    */
   memberTemplate() {
     return html`
-      <div class="py-1 px-4 truncate" role="menuitem">
-        <section class="mt-1">
-          <!-- Name -->
-          <span class="block text-xl font-bold text-gray-900 truncate"
-            >${this.member.name}</span
-          >
-
-          <!-- Title -->
-          <span class="block text-sm text-gray-500 truncate"
-            >${this.member.jobTitle}</span
-          >
-
-          <!-- Organization -->
-          ${
-            !!this.member.organization
+      <div class="px-4 py-5 w-80 truncate">
+        <section class="flex gap-4">
+          <!-- Avatar -->
+          ${this.member.avatarUrl &&
+          html`<img
+            alt=""
+            class="w-14 h-14 rounded-full"
+            src="${this.member.avatarUrl}"
+          />`}
+
+          <div class="flex flex-col">
+            <!-- Name -->
+            <span
+              class="block text-lg font-semibold leading-5 text-gray-900 truncate"
+              >${this.member.name}</span
+            >
+
+            <!-- Title -->
+            <span class="block text-sm leading-5 text-gray-500 truncate"
+              >${this.member.jobTitle}</span
+            >
+
+            <!-- Organization -->
+            ${!!this.member.organization
               ? html`
-            <div class="flex flex-row justify-start items-center mt-1">
-              ${
-                this.member.organization.logo_url &&
-                html`<img
-                  class="mr-1 w-5 h-5"
-                  src="${this.member.organization.logo_url}"
-                />`
+                  <div
+                    class="flex flex-row justify-start items-center mt-1 leading-5"
+                  >
+                    ${this.member.organization.logo_url &&
+                    html`<img
+                      alt=""
+                      class="mr-1 w-5 h-5"
+                      src="${this.member.organization.logo_url}"
+                    />`}
+                    <!-- If organisation doesn't include a protocol (ie https://), add one so it's treated as absolute -->
+                    <a
+                      href="${this.member.organization.website.match(
+                        /https?:\/\//
+                      )
+                        ? this.member.organization.website
+                        : `https://${this.member.organization.website}`}"
+                      target="_blank"
+                      rel="noreferrer"
+                      class="mr-2 text-sm font-semibold text-[#6C4DF6] hover:underline"
+                      >${this.member.organization.name}</a
```

Comment:
It looks like this one overflows if the org name is too long (apologies for the heavy-handed blur, you can reproduce locally by editing in the console or changing the org name in Orbit)

![CleanShot 2023-05-31 at 17 20 23@2x](https://github.com/orbit-love/orbit-browser-extension/assets/2587348/e8edbea4-d4cb-4df8-a345-edba7f3bfd41)

Reply:
<img width="325" alt="CleanShot 2023-05-31 at 16 46 16@2x" src="https://github.com/orbit-love/orbit-browser-extension/assets/45462299/7c631435-596c-4cd0-ac15-9b122967bc2b">

Fixed!